docs/troubleshooting.md

```diff
@@ -0,0 +1,255 @@
+
+```bash
+$ kubectl delete clusterserviceversion jaeger-operator.v1.8.2 -n local-operators
+```
+
+Edit the installation `CatalogSourceConfig` and modify the `spec.packages` field to remove the operator's packageName with the following command:
+
+```bash
+$ kubectl edit catalogsourceconfig installed-upstream-community-operators -n marketplace
+```
+
+This will open up your terminal's default editor. Look at the `spec` section of the `CatalogSourceConfig`:
+
+```bash
+spec:
+  targetNamespace: local-operators
+  packages: descheduler,jaeger
+  csDisplayName: "Upstream Community Operators"
+  csPublisher: "Red Hat"
+```
+
+Remove `jaeger` from `spec.packages`:
+
+```bash
+spec:
+  csDisplayName: Community Operators
+  csPublisher: Community
+  packages: descheduler
+```
+
+Save the change and the marketplace-operator will reconcile the `CatalogSourceConfig`.
+
+If only one operator is installed into the `CatalogSourceConfig`, delete the installation `CatalogSourceConfig`. If created via the OpenShift OperatorHub UI, it will be named `installed-<OPERATORSOURCE>-<TARGETNAMESPACE>` based on the `OperatorSource` that the operator comes from and the `targetNamespace` that the operator was installed to.
+
+```bash
+$ kubectl delete catalogsourceconfig installed-upstream-community-operators -n marketplace
+```
+
+### Authenticating Private App-Registry Repositories
+
+If you have an app-registry repository that is backed by authentication, you can specify an authentication token in a Secret. To do this, create a Secret in the same namespace as your Operator Source:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: marketplacesecret
+  namespace: openshift-marketplace
+type: Opaque
+stringData:
+    token: "basic yourtokenhere=="
+```
+
+Then, to associate that secret with a registry, simply add a reference to the secret in the OperatorSource spec:
+
+```yaml
+apiVersion: "operators.coreos.com/v1"
+kind: "OperatorSource"
+metadata:
+  name: "certified-operators"
+  namespace: "openshift-marketplace"
+  labels:
+    opsrc-provider: certified
+spec:
+  type: appregistry
+  endpoint: "https://quay.io/cnr"
+  registryNamespace: "certified-operators"
+  displayName: "Certified Operators"
+  publisher: "Red Hat"
+  authorizationToken:
+    secretName: marketplacesecret
+```
+
+When downloading the repository from the app-registry, Marketplace will pass along the authentication token.
+
+## Common Failures
+
+### No packages show up in the UI
+
+When you install an Openshift cluster, the embedded Marketplace (OperatorHub) comes with a default list of operators that can be installed onto the cluster. A number of resources need to be investigated if however you see a message like this: 
+
+```
```

Comment:
@galletti94 I tried exploring ways to insert an image in a .md file. Doesn't look like there's a clean way to do it. Besides, have the error typed fight make it easier for the reader to CTRL+F keywords from the error message.

Reply:
@anik120 

Disagree. From the github docs:

https://help.github.com/en/articles/editing-wiki-content#adding-images

Reply:
@kevinrizza Notice that all examples of inserting images in a .md file in the github docs (or otherwise) uses hosted images. I found one example where the image was stored in a folder in the repository (say images), and the image was rendered using `![](images/<name-of-file>.png)`. However, that did not seem to do the trick. 
We could insert the image with this PR, and later use the hosted github link (https//github.com/operator-framework/operator-marketpalce/images/<image-name>...) for a separate PR to edit this doc. 
Or I could edit this PR to include what I mentioned above (`![]images/<name-of-file>.png`) and explore why it's not rendering correctly. 
We could explore the option of hosting the image somewhere else and include that link here but I don't suppose we'd want to depend on anything external for a screenshot.

Reply:
What's wrong with hosting the image in a subfolder of the docs folder?

Reply:
@galletti94 @kevinrizza seems like putting the images folder inside the doc folder did the trick. Screenshot added.